client/packages/programs/src/VaccineCourseEditModal/VaccineCourseEditModal.tsx

```diff
@@ -380,7 +380,12 @@ const AgeCell = (props: CellProps<VaccineCourseDoseFragment>) => {
       {...props}
       units={[
         { key: 'year', ratio: 12, label: t('label.years-abbreviation') },
-        { key: 'month', ratio: 1, label: t('label.months-abbreviation') },
+        {
+          key: 'month',
```

Comment:
I wonder if we should put a max on age too~....





... though there could be secret 1,000 yo immortals among us 👀

Reply:
Haha who are we to stop someone wanting to use OMS to vaccinate immortal beings 🙇 

Yeah I wondered, but not sure what age to use...150?? Also, it's only configured centrally, so probably safe to expect sensible ages?

Reply:
Oldest person is 122... so maybe 150 fine..? 👀  idk